OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -161,6 +182,67 @@ def setParameterNode(self, inputParameterNode: Optional[OpenLIFUTransducerTracke
     def onDataParameterNodeModified(self, caller, event) -> None:
         self.updateApproveButton()
         self.updateApprovalStatusLabel()
+        self.updateInputOptions()
+        
+    @vtk.calldata_type(vtk.VTK_OBJECT)
+    def onNodeRemoved(self, caller, event, node : slicer.vtkMRMLNode) -> None:
+        """ Update volume and photoscan combo boxes when nodes are removed from the scene"""
+        self.updateInputOptions()
+
+    @vtk.calldata_type(vtk.VTK_OBJECT)
+    def onNodeAdded(self, caller, event, node : slicer.vtkMRMLNode) -> None:
+        """ Update volume and photoscan combo boxes when nodes are added to the scene"""
+        self.updateInputOptions()
+
+    def updateInputOptions(self):
+        """Update the algorithm input options"""
+        self.algorithm_input_widget.update()
+
+        # Determine whether transducer tracking can be run based on the status of combo boxes
+        self.checkCanRunTracking()
+
+    def onLoadPhotoscanClicked(self):
+        # Using a custom file dialog instead of slicer.util.openAddModelDialog() incase database specific 
+        # customizations are required later 
+        qsettings = qt.QSettings()
+
+        filepath: str = qt.QFileDialog.getOpenFileName(
+            slicer.util.mainWindow(), # parent
+            'Load photoscan', # title of dialog
+            qsettings.value('OpenLIFU/databaseDirectory','.'), # starting dir, with default of '.'
+            "Model (*.obj; *.vtk);;All Files (*)", # file type filter
+        )
```

Comment:
I am not able to replicate this issue since it filters correctly on Windows. But I removed the semicolon separating the file types since that it how it is implemented in the Data module. I'm assuming that's related to why the obj files aren't showing up for you.

Reply:
(that fix worked, thanks!)